src/functions/module_lookup.ts

```diff
@@ -848,13 +856,18 @@ export const PIVOT = {
       _t("Whether to include the column titles or not.")
     ),
     arg("column_count (number, optional)", _t("number of columns")),
+    arg(
+      "include_measures (boolean, default=TRUE)",
+      _t("Whether to include the measures row or not.")
```

Comment:
Just "measure" could be understood as measure values ?
```suggestion
      "include_measure_titles (boolean, default=TRUE)",
      _t("Whether to include the measure titles row or not.")
```